src/lib.rs

```diff
@@ -2,28 +2,37 @@ use geo::{LineString, Point, Polygon};
 use std::default::Default;
 
 // See https://stackoverflow.com/questions/24047686
-#[derive(Debug)]
-pub struct Params {
-    n_circles: usize,
-    distances: Vec<f64>,
-}
+// #[derive(Debug)]
+// pub struct Params {
+//     n_circles: usize,
+//     distances: Vec<f64>,
+// }
 
-impl Default for Params {
-    fn default() -> Self {
-        // todo: distances should be:
-        // zonebuilder::zb_100_triangular_numbers
-        // 1    3    6   10   15   21   28   36   45   55   66 ...
-        Params { n_circles: 5, distances: 10.0}
-    }
+// impl Default for Params {
+//     fn default() -> Self {
+//         // todo: distances should be:
+//         // zonebuilder::zb_100_triangular_numbers
+//         // 1    3    6   10   15   21   28   36   45   55   66 ...
+//         Params { n_circles: 5, distances: 10.0}
+//     }
+// }
+
+// Trying again
+// https://doc.rust-lang.org/std/default/trait.Default.html
+#[derive(Default)]
+struct SomeOptions {
+    n_circles: usize 5,
```

Comment:
This isn't the syntax. Check out https://doc.rust-lang.org/rust-by-example/custom_types/structs.html

Reply:
Updated. Now reads:

```
#[derive(Debug)]
pub struct Params {
    n_circles: usize,
    num_segments: usize,
    distances: Vec<f64>,
}

// https://doc.rust-lang.org/std/default/trait.Default.html

impl Default for Params {
    fn default() -> Self {
        // todo: distances should be:
        // zonebuilder::zb_100_triangular_numbers
        // 1    3    6   10   15   21   28   36   45   55   66 ...
        Params { n_circles: 5, num_segments: 12, distances: vec![1.0, 3.0, 6.0, 10.0, 15.0]}
    }
}
```